### ISSUE TITLE
Splash Screen/Preloader Update

### ISSUE THREAD

**Issue by smaranjitghose** [smaranjitghose/doc2pen#1055]
[Reference](https://codepen.io/CameronFitzwilliam/details/JrGxxr)

## Modifications (w.r.t the reference):

- The color of the vector should correspond to the theme of the website
- The text doc2pen should be below the vector

**Comment by pallavi-1812** [comment]
Hello, I would like to work on this issue but I have a doubt. We are supposed to add the same vector image given in reference in the splash screen right?

**Comment by smaranjitghose** [comment]
No. You are expected to modify the colors

**Comment by pallavi-1812** [comment]
Yeah. I would like to work on this issue. Can you assign it to me?

**Comment by tusharnankani** [comment]
@pallavi-1812 Any updates?

**Comment by pallavi-1812** [comment]
No, I am getting errors in setting up the project. It's fine if you want to assign it to someone else.

**Comment by vedant-jain03** [comment]
I want to work on this issue.

**Comment by nik464** [comment]
can you assign me under lgm soc

**Comment by nik464** [comment]
please

**Comment by nik464** [comment]
it will be helpful

**Comment by smaranjitghose** [comment]
Please don't spam with multiple comments

**Comment by anushbhatia** [comment]
@nik464 do you wan to take this up?
